.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Please fill the `[ ]` with an `x` to make sure that you read and followed the checklist
+
+- [ ] I have checked that the proposed addition is not already in the awesome list
+- [ ] I have signed-off my commits as detailed [here](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md)
+- [ ] The links follow the link template:
+
+        * [Title](URL) - Description.
+- [ ] The addition is correctly classified as a paper, blog post, talk, software or dataset.
```

Comment:
all points should have a dot in the end or remove all of them to have consistent style (I prefer first option)